### PR TITLE
Update documentation for Name Site Entry route

### PR DESCRIPTION
**Pull request by chromium-52**

### CODE REVIEW

**Review by Tvcz — APPROVED** [review]

File: docs/sftt/sftt-api-spec.md [hunk 1]
```diff
@@ -2261,7 +2261,7 @@ Site entry successfully named.
 
 ##### `400 BAD REQUEST`
 
-If the `site_id` specified does not exist.
+If the `site_id` specified does not exist or if `name` is more than 60 characters long.
```
Comment (SofieDunt) [inline docs/sftt/sftt-api-spec.md:2264]: 👍

**Review by SofieDunt — COMMENTED** [review]
Was the null initially to be able to delete a name? Would the empty string also be a bad request?

**Comment by Tvcz** [comment]
null was never accepted, the documentation was incorrect. Now an empty string resets the name.

**Comment by Tvcz** [comment]
Come to think of it that behavior should probably be documented, I can make another commit for it later.

**Review by SofieDunt — APPROVED** [review]

**Pull request merged**